core-models/src/main/scala/com/mesosphere/usi/core/models/Resources.scala

```diff
@@ -0,0 +1,10 @@
+package com.mesosphere.usi.core.models
+
+/**
+  * Resources that were reserved with a reservation
```

Comment:
Not sure if that makes sense. Isn't this a [Mesos resource](http://mesos.apache.org/documentation/attributes-resources/)? Should we reference them?